pkg/rancher-desktop/components/RdSelect.vue

```diff
@@ -27,6 +27,18 @@ export default Vue.extend({
         this.$emit('input', newValue);
       },
     },
+    /**
```

Comment:
Shouldn't this be part of a separate PR?

Reply:
Yes, not sure why it is include; looks like it needs a rebase.